### PR TITLE
Add splitter

### PR DESCRIPTION
**Pull request by kruglikovstepan**

### CODE REVIEW

**Review by nightskylark — DISMISSED** [review]

File: js/ui/gantt/ui.gantt.js [hunk 1]
```diff
@@ -25,6 +37,12 @@ class Gantt extends Widget {
             .appendTo(this._$treeListWrapper);
         this._$splitter = $("<div>")
             .addClass(GANTT_SPLITTER_CLASS)
+            .addClass(GANTT_SPLITTER_TRANSPARENT_CLASS)
+            .css('left', this.option("treeListWidth"))
```
Comment (DokaRus) [inline js/ui/gantt/ui.gantt.js:42]: Will it work correctly with the RTL mode?

File: js/ui/gantt/ui.gantt.js [hunk 2]
```diff
@@ -118,10 +177,16 @@ class Gantt extends Widget {
...
-    _updateWidth() {
-        const treeListWidth = this.option("treeListWidth");
+    _updateWidth(treeListWidth) {
         this._$treeListWrapper.width(treeListWidth);
         this._$treeList.width(treeListWidth);
+        this._$splitter.css('left', treeListWidth);
+        this._ganttView && this._ganttView._setWidth(this.$element().width() - treeListWidth - GANTT_SPLITTER_BORDER_WIDTH);
+    }
+
+    _dispose() {
```
Comment (DokaRus) [inline js/ui/gantt/ui.gantt.js]:
```suggestion
    _clean() {
```
`instance.repaint()` => `instance._clean()` + `instance._render()`
`instance.dispose()` => `instance._dispose()` (also calls the `_clean` method)

File: styles/widgets/material/gantt.material.less [hunk 3]
```diff
@@ -121,6 +130,31 @@
...
+
+    .dx-gantt-task-edit-wrapper-successor {
+        padding-top: 8px;
+    }
+
+    .dx-gantt-task-edit-dependency-r,
+    .dx-gantt-task-edit-successor-dependency-r,
+    .dx-gantt-task-edit-dependency-l,
+    .dx-gantt-task-edit-successor-dependency-l {
+        background: white;
```
Comment (DokaRus) [inline styles/widgets/material/gantt.material.less]: Should we move it to the color constants?

**Review by DokaRus — APPROVED** [review]
ToDo:
1. Research - perhaps we can use [Resizable](https://js.devexpress.com/Documentation/ApiReference/UI_Widgets/dxResizable/) component
2. Base RTL support

**Pull request merged**